### PR TITLE
Add alias for urls with hyphens

### PR DESCRIPTION
**Pull request by letzya**
Added aliases: 
1. the actual automatic redirect, i.e url with 2 hyphens `- /tyk-cloud/teams--users`, `/tyk-cloud/troubleshooting--support/faqs` etc.
2. a new alias with one hyphen so we can use it throughout the docs `- /tyk-cloud/teams-and-users`,  `/tyk-cloud/troubleshooting-and-support/faqs` etc

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/tyk-cloud/teams-&-users.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ menu:
   main:
     parent: "Tyk Cloud"
 aliases:
-  - /tyk-cloud/teams-&-users
+  - /tyk-cloud/teams-and-users
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/teams-&-users.md]:
```suggestion
  - /tyk-cloud/teams-&-users
  - /tyk-cloud/teams-and-users
```

File: tyk-docs/content/tyk-cloud/troubleshooting-&-support/faqs.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ menu:
   main:
     parent: "Troubleshooting & Support"
 aliases:
-  - tyk-cloud/troubleshooting-&-support/faqs
+  - /tyk-cloud/troubleshooting--support/faqs/
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/troubleshooting-&-support/faqs.md]:
```suggestion
  - tyk-cloud/troubleshooting-&-support/faqs
  - /tyk-cloud/troubleshooting--support/faqs
```

File: tyk-docs/content/tyk-cloud/troubleshooting-&-support/faqs.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ menu:
     parent: "Troubleshooting & Support"
 aliases:
   - tyk-cloud/troubleshooting-&-support/faqs
+  - /tyk-cloud/troubleshooting--support/faqs
+  - /tyk-cloud/troubleshooting-and-support/faqs/
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/troubleshooting-&-support/faqs.md]:
```suggestion
  - /tyk-cloud/troubleshooting-and-support/faqs
```

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md [hunk 4]
```diff
@@ -8,7 +8,8 @@ menu:
     parent: "Environments & Deployments"
 weight: 6
 aliases:
-  - tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments--deployments/managing-apis
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md]:
```suggestion
  - tyk-cloud/environments-&-deployments/managing-apis
  - /tyk-cloud/environments--deployments/managing-apis
```

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md [hunk 5]
```diff
@@ -8,7 +8,9 @@ menu:
     parent: "Environments & Deployments"
 weight: 6
 aliases:
-  - tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments--deployments/managing-apis
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md]:
```suggestion
  - /tyk-cloud/environments-deployments/managing-apis
```

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md [hunk 6]
```diff
@@ -8,7 +8,9 @@ menu:
     parent: "Environments & Deployments"
 weight: 6
 aliases:
-  - tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments-deployments/managing-apis
+  - /tyk-cloud/environments-deployments/managing-apis
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md:11]:
```suggestion
```

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md [hunk 7]
```diff
@@ -8,7 +8,8 @@ menu:
     parent: "Environments & Deployments"
 weight: 6
 aliases:
-  - tyk-cloud/environments-&-deployments/managing-apis
+  - /tyk-cloud/environments-&-deployments/managing-apis
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-apis.md]:
```suggestion
```

**Comment by dcs3spp** [comment]
closing

**Pull request closed without merging**